src/pages/partners.jsx

```diff
@@ -0,0 +1,9 @@
+const partners = () => {
+    return (
+        <div>
+            
+        </div>
+    )
+}
+
+export default partners
```

Comment:
@pranav-more  kindly rename this *Function Name* not File to `Partners` instead of `partners`

As React only recognises functions that start with a capital letter as a Component

- Thanks

Reply:
done !